frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-unused-vars, no-undef */
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { K8sKind, k8sList, referenceForModel } from '../../module/k8s';
+import { ResourceLink, resourcePathFromModel } from '../utils/resource-link';
+
+const globalConfigListStateToProps = ({k8s}) => ({
+  configResources: k8s.getIn(['RESOURCES', 'configResources']),
+});
+
+const ItemRow = ({item}) => {
+  return <div className="row co-resource-list__item">
+    <div className="col-sm-5 col-xs-6">
+      <ResourceLink kind={referenceForModel(item.model)} name={item.metadata.name} />
+    </div>
+    <div className="col-sm-5 col-xs-6">
+      {item.kind}
+    </div>
+    <div className="col-sm-2 hidden-xs">
+      <Link to={`${resourcePathFromModel(item.model)}/${item.metadata.name}/yaml`} className="btn btn-default pull-right">Edit YAML</Link>
+    </div>
+  </div>;
+};
+
+class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps> {
+  state = {
+    errorMessage: '',
+    items: [],
+  }
+
+  componentDidMount() {
+    Promise.all(this.props.configResources.map(model => {
```

Comment:
Why are we using `k8sList` instead of a `<Firehose>`?

Reply:
It will be a dozen or more resources, and we didn't want to open so many websockets. The contents of the table shouldn't change anyway.

Reply:
> we didn't want to open so many websockets

This shouldn't be a concern, [modern browsers can open many WebSockets](https://cs.chromium.org/chromium/src/net/socket/client_socket_pool_manager.cc?q=WEBSOCKET_SOCKET_POOL&sq=package:chromium&type=cs&l=29). Plus it would make the component stateless.

Reply:
What happens if you are viewing these resources as the cluster is updating?